app/src/main/res/values/strings.xml

```diff
@@ -199,22 +199,22 @@
     <string name="center_text_desc">Edit text in center of the screen</string>
     <string name="clipboard">Clipboard</string>
     <string name="existing_document">Select existing document</string>
-    <string name="open_source_licenses">Open Source Licenses</string>
+    <string name="open_source_licenses">Libre Software Licenses</string>
```

Comment:
Open Source Licenes is just fine, and I don't want to start a discussion on oss/foss/floss/free-sw :D.